scripts/declarations/utils/index.js

```diff
@@ -33,48 +36,55 @@ export default class DeclarationUtils {
     return modifiedServicesIds;
   }
 
-  async getModifiedServiceTermsTypes() {
-    const { modifiedFilePaths, modifiedServicesIds } = await this.getModifiedData();
+  async getModifiedServicesTermsTypes() {
+    const { modifiedFilePaths } = await this.getModifiedData();
     const servicesTermsTypes = {};
 
     await Promise.all(modifiedFilePaths.map(async modifiedFilePath => {
-      const serviceId = DeclarationUtils.filePathToServiceId(modifiedFilePath);
+      const serviceId = DeclarationUtils.getServiceIdFromFilePath(modifiedFilePath);
```

Comment:
Extract the block in `map` to new function, it is too long to be declared this way in an iteration 🙂

Reply:
I do not understand what you mean here

Reply:
The whole anonymous function that is defined and called as the `map` parameter is huge. I believe it would help with readability if it was extracted into its own named function.

Reply:
With the code simplification, I no longer find it useful.